spec/support/examples/idempotent_resource.rb

```diff
@@ -1,9 +1,9 @@
 shared_examples 'an idempotent resource' do
   it 'should run without errors' do
-    apply_manifest(@manifest, :expect_changes => true)
+    apply_manifest(@manifest, :catch_failures => true)
   end
 
   it 'should run a second time without changes' do
-    apply_manifest(@manifest, :expect_changes => false)
+    apply_manifest(@manifest, :catch_changes => true)
```

Comment:
This check for idempotency was broken previously and errors were not being raised, even if changes were made on the 2nd run of the given manifest.